examples/Learning_example/learning_example.ipynb

```diff
@@ -0,0 +1,2295 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #48.        def merge_model(self, rx_model):
</pre><p>what is rx? maybe spell out?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>